README.md

```diff
@@ -14,9 +14,9 @@ In debug mode, `strong-error-handler` returns full error stack traces and intern
 
 ## Supported versions
 
-Current|Long Term Support|Maintenance
-:-:|:-:|:-:
-3.x|2.x|1.x
+| Current | Long Term Support | Maintenance |
+| :-----: | :---------------: | :---------: |
+|   3.x   |        2.x        |     1.x     |
```

Comment:
Since we are going to publish this PR as `4.0.0`, we need to update this table too. Please use the format we have in the main loopback repository (see https://github.com/strongloop/loopback#module-long-term-support-policy), were each version line has it's own table row and the table has the following columns: Version, Status, Published, EOL